app/controllers/csvs_controller.rb

```diff
@@ -0,0 +1,70 @@
+class CsvsController < ApplicationController
+
+  def index
+    respond_to do |format|
+      @csv_exports = CsvExport.where(user_id: current_user.id)
+      format.html
+      format.json { render json: @csv_exports }
+    end
+  end
+  
+  def show
+    respond_to do |format|
+      @csv_export = CsvExport.find_by_id(params[:id])
```

Comment:
we can use an actual `find` and then do `rescue_from ActiveRecord::RecordNotFound` and render custom json there